pkg/app/piped/cloudprovider/lambda/BUILD.bazel

```diff
@@ -4,21 +4,21 @@ go_library(
     name = "go_default_library",
     srcs = [
         "client.go",
-        "lambda.go",
         "function.go",
+        "lambda.go",
     ],
     importpath = "github.com/pipe-cd/pipe/pkg/app/piped/cloudprovider/lambda",
     visibility = ["//visibility:public"],
     deps = [
         "//pkg/config:go_default_library",
-        "@io_k8s_sigs_yaml//:go_default_library",
         "@com_github_aws_aws_sdk_go//aws:go_default_library",
         "@com_github_aws_aws_sdk_go//aws/awserr:go_default_library",
         "@com_github_aws_aws_sdk_go//aws/credentials:go_default_library",
         "@com_github_aws_aws_sdk_go//aws/credentials/ec2rolecreds:go_default_library",
         "@com_github_aws_aws_sdk_go//aws/ec2metadata:go_default_library",
         "@com_github_aws_aws_sdk_go//aws/session:go_default_library",
         "@com_github_aws_aws_sdk_go//service/lambda:go_default_library",
+        "@io_k8s_sigs_yaml//:go_default_library",
```

Comment:
nit: I couldn't figure out why does this file be changed 😂

Reply:
`make gazelle` does it. Basically, It adds missing ones and orders them in its convention.

Reply:
I see, thank you 👍